backend/models/class.model.ts

```diff
@@ -0,0 +1,35 @@
+import mongoose, { Schema, Document, model } from "mongoose";
+
+export interface Class extends Document {
+  class_name: string;
```

Comment:
```suggestion
  className: string;
```
can you update all the fields to be camelCase?